templates/defaults.cfg

```diff
@@ -49,7 +49,7 @@ defaults
     option {{ option }}
 {% endfor -%}
 {% endif -%}
-{% if ansible_distribution != 'CentOS' %}
+{% if ansible_distribution != 'CentOS' and ansible_distribution != 'Alpine' %}
```

Comment:
Alpine does not ship the error files. So, I figured they should not be included in the default config for Alpine.